pages/apply/index.tsx

```diff
@@ -38,7 +38,7 @@ export default function ApplicationHomePage(): JSX.Element {
         housing offer is made.
       </Paragraph>
 
-      <Announcement variant="info">
+      <Announcement variant="success">
```

Comment:
The design looks to be a faded yellow, you can override with some custom styles if not available in design system.

Reply:
I've confirmed with Chris that the background color needs to be blue as there were two versions of the design